server/api.go

```diff
@@ -807,6 +807,8 @@ func (a *API) updateWhitelist(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	a.p.API.LogWarn("Whitelist updated", "size", len(ids))
```

Comment:
shouldn't this be an info?

Reply:
Yes, bad copy-paste. Thanks for catching!